utils/postinstall.sh

```diff
@@ -32,3 +32,9 @@ elif type "wget" > /dev/null 2>&1; then
     wget -T 2 -q -O /dev/null "$REPO_URL" || true
     exit 0
 fi
+
+if [ ! -f /etc/containerlab/suid_setup_done ]; then
+    groupadd -r clab_admins
+    usermod -aG clab_admins "$SUDO_USER"
+    touch /etc/containerlab/suid_setup_done
```

Comment:
@vista- I wonder, maybe instead of creating an empty file in /etc/containerlab we can check that the `clab_admins` group exists?

Reply:
That wouldn't be an equivalent operation -- if someone wants to let all users use Containerlab, they can simply delete the group and the check is skipped. This way, they would have to delete the group after every upgrade.